pootle/core/views.py

```diff
@@ -632,6 +634,9 @@ def get_context_data(self, *args, **kwargs):
         ctx, cookie_data = self.sidebar_announcements
         ctx.update(
             super(PootleBrowseView, self).get_context_data(*args, **kwargs))
+
+        language_code, project_code = split_pootle_path(self.pootle_path)[:2]
```

Comment:
PootleBrowseView (well PootleDetailView actually) has canonical methods for getting lang and project - i think this should use those, ie
```
lang_code = self.language.code if self.language else ""
```
```